### DISCUSSION TITLE
Support for gpu codec plugin

### DISCUSSION THREAD

**Discussion by eecoder-dyf** [rustdesk/rustdesk#7552] [General]
The [Sunshine](https://github.com/LizardByte/Sunshine) can achieve high fps encoding using gpu codec provided by NIVIDIA, Intel and AMD officially. Is this possible for rustdesk?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/5264

**Reply by eecoder-dyf** [reply]
Well, it seems not support Linux yet, hope for new update!
